articles/storage/blobs/object-replication-overview.md

```diff
@@ -108,6 +108,9 @@ The source and destination containers must both exist before you can specify the
 
 You can call the [Set Blob Tier](/rest/api/storageservices/set-blob-tier) operation on a blob in the destination container to move it to the archive tier. For more information about the archive tier, see [Hot, Cool, and Archive access tiers for blob data](access-tiers-overview.md#archive-access-tier).
 
+> [!NOTE]
```

Comment:
Hi @Amrinder-Singh29 - question about this note - I assume that this behavior applies only to blobs already replicated. Changing the source tier **after** the blob has been replicated to the destination won't change the tier of the blob at the destination correct? I assume that if you change a blob's tier before it is replicated, then that blob copied to the destination will have match the tier of the source. Seems like we might want to make that clear. Thoughts?

Reply:
Hi @normesta ,

For the first part, it is correct. Once the blob has been replicated and then the access tier on the source is changed, it won't have any change on the access tier of the destination. 

For the second part, it is not correct. The access tier of blob in the destination shall depend on the access tier of the account and is inferred from there. Whether a blob access tier gets changed prior to blob getting replicated or after, it doesn't affect access tier on the destination.

Hope that helps.